src/components/Dashboard/StaffNotation.vue

```diff
@@ -86,83 +86,78 @@
-              >
-                <v-list-item-title>
-                  <span class="app-blue">Put Back On</span>
-                </v-list-item-title>
-              </v-list-item>
-            </template>
-
-            <template v-if="!isHistorical">
-              <v-list-item
-                data-type="admin-freeze"
-                @click="showAdministerFreezeDialog()">
-                <v-list-item-title>
-                  <span class="app-blue">{{ isAdminFreeze ? 'Unfreeze Business' : 'Freeze Business' }}</span>
-                </v-list-item-title>
-              </v-list-item>
-            </template>
-
-            <template v-if="isBenBcCccUlc && !isHistorical">
-              <v-list-item
-                data-type="consent-continue-out"
-                @click="goToConsentContinuationOutFiling()"
-                :disabled="disabled || isAdminFreeze"
-              >
-                <v-list-item-title>
-                  <span class="app-blue">Consent to Continuation Out</span>
-                </v-list-item-title>
-              </v-list-item>
-            </template>
+            <v-list-item
+              v-if="!isHistorical"
+              data-type="administrative-dissolution"
+              @click="showAdministrativeDissolutionDialog()"
+              :disabled="!isAllowed(AllowableActions.ADMINISTRATIVE_DISSOLUTION)"
+            >
+              <v-list-item-title>
+                <span class="app-blue">Administrative Dissolution</span>
+              </v-list-item-title>
+            </v-list-item>
+
+            <v-list-item
+              v-if="isAllowed(AllowableActions.RESTORATION)"
+              data-type="restoration"
+              @click="goToRestorationFiling(ApplicationTypes.CREATE_UI, FilingSubTypes.RESTORATION_FULL)"
+            >
+              <v-list-item-title>
+                <span class="app-blue">Restore Company</span>
+              </v-list-item-title>
+            </v-list-item>
+
+            <v-list-item
+              v-if="isAllowed(AllowableActions.PUT_BACK_ON)"
+              data-type="put-back-on"
+              @click="showPutBackOnDialog()"
+            >
+              <v-list-item-title>
+                <span class="app-blue">Put Back On</span>
+              </v-list-item-title>
+            </v-list-item>
+
+            <v-list-item
+              v-if="isAllowed(AllowableActions.FREEZE_UNFREEZE)"
+              data-type="admin-freeze"
+              @click="showAdministerFreezeDialog()"
+            >
+              <v-list-item-title>
+                <span class="app-blue">{{ isAdminFreeze ? 'Unfreeze Business' : 'Freeze Business' }}</span>
+              </v-list-item-title>
+            </v-list-item>
 
             <v-list-item
-              v-if="isInLimitedRestoration"
+              data-type="consent-continue-out"
+              @click="goToConsentContinuationOutFiling()"
+              :disabled="!isAllowed(AllowableActions.CONSENT_CONTINUATION_OUT)"
```

Comment:
I don't know if this should be shown in all cases (eg, historical?) but we can change this easily in the future as we get UX feedback.